### DISCUSSION TITLE
Getting 404 On All Links (Static Site Built with .md, .scss, .html)

### DISCUSSION THREAD

**Discussion by jrwesa** [community/community#119996] [Programming Help]
### Select Topic Area

Question

### Body

# What Am I Doing Wrong?

**Problem**: All of my links are broken. Whether I am linking to another page (.md file) or linking to another header (#, ##, etc.), I get a 404 message that reads: 

![image](https://github.com/community/community/assets/165824176/2fde8585-37b2-45e3-99d9-f8f5cc5f00eb)

See Links for Broken Links: https://jrwesa.github.io/docs/layout/subcategory/child-1/

**Site**: Built with the [just-the-docs](https://github.com/just-the-docs/just-the-docs) theme and .md, .scss files. 
- Repo: https://github.com/jrwesa/jrwesa.github.io/tree/main
- Site URL: https://jrwesa.github.io/

**NOTE**: Currently, my GitHub Pages site deploys. The navigation seen on the left side of the site reflects my markdown pages.

I had read other threads, but many people are dealing with their site not deploying _at all_. (Seemingly a problem with their index file, but I'm not sure that is my problem)

**Comment by LiteBrite82** [comment]
I'm not familiar with just-the-docs; so take my advice with a grain of salt. 

Try changing index.md to index.html (I saw this in the console, it looks like the browser doesn't like that the doctype is html, but the file is .md

It could be unrelated though; my first instinct is to look for routing issues. 

<img width="1503" alt="Screenshot 2024-04-17 at 3 08 29 PM" src="https://github.com/community/community/assets/163985914/1788bd00-5708-43af-a77a-0cb17111242a">

**Reply by LiteBrite82** [reply]
Wait, no... I browsed through the documentation... I don't think thats it... 

Im gonna send this over to the Programming Help channel. this seems more like an issue with the just-the-docs configuration and build. This will get much more attention over there.

**Reply by jrwesa** [reply]
Thanks for investigating!
